secretsmanager/src/aes.cpp

```diff
@@ -21,22 +21,19 @@
 namespace concord::secretsmanager {
 using std::vector;
 using std::string;
-using std::unique_ptr;
 using concord::crypto::openssl::OPENSSL_SUCCESS;
 using concord::crypto::openssl::UniqueCipherContext;
 
-vector<uint8_t> AES_CBC::encrypt(const string& input) {
+vector<uint8_t> AES_CBC::encrypt(std::string_view input) {
   if (algo_ == concord::crypto::SignatureAlgorithm::EdDSA) {
     if (input.empty()) {
       return {};
     }
 
-    unique_ptr<unsigned char[]> ciphertext(new unsigned char[input.size() + AES_BLOCK_SIZE]);
-    unique_ptr<unsigned char[]> plaintext(new unsigned char[input.size() + 1]);
+    auto ciphertext = std::make_unique<unsigned char[]>(input.size() + AES_BLOCK_SIZE);
```

Comment:
Instead of allocating both an array and a vector, allocate a vector here and return it.